docs/commit-guide.md

```diff
@@ -0,0 +1,131 @@
+# Commit Messages Guide
+
+This guide describes how we adding human and machine readable meaning to commit messages.
+
+Refer to [Conventional Commits 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/)
+
+## Summary
+
+The Conventional commits specification is a lightweight convention on top of commit message. It provides an easy set of rules for creating an explicit commit history, which dedicated to achieving better readability, velocity and automation.
```

Comment:
Conventional Commits

Reply:
which dedicated to achieving better readability, velocity and automation -> which improves better readability ...